include/bluetooth/mesh/sensor_types.h

```diff
@@ -44,6 +44,336 @@ extern "C" {
 		const struct bt_mesh_sensor_type *volatile __unused val =      \
 			&_type;                                                \
 	}
+
+/** @defgroup bt_mesh_sensor_formats_percentage Percentage sensor formats
+ *  @{
+ */
+
+/** Percentage 8
+ *  - Unit: Percent
+ *  - Encoding: 8 bit unsigned scalar (Resolution: 0.5)
+ *  - Range: 0 to 100.0
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_percentage_8;
+
+/** Percentage 16
+ *  - Unit: Percent
+ *  - Encoding: 16 bit unsigned scalar (Resolution: 0.01)
+ *  - Range: 0 to 100.0
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_percentage_16;
+
+/** Percentage delta trigger
+ *  - Unit: Percent
+ *  - Encoding: 16 bit unsigned scalar (Resolution: 0.01)
+ *  - Range: 0 to 655.35
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_percentage_delta_trigger;
+/** @} */
+
+/** @defgroup bt_mesh_sensor_formats_environmental Environmental sensor formats
+ *  @{
+ */
+
+/** Temp 8
+ *  - Unit: Celsius
+ *  - Encoding: 8 bit signed scalar (Resolution: 0.5)
+ *  - Range: -64.0 to 63.0
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_temp_8;
+
+/** Temp
+ *  - Unit: Celsius
+ *  - Encoding: 16 bit signed scalar (Resolution: 0.01)
+ *  - Range: -327.67 to 327.67
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_temp;
+
+/** Co2 concentration
+ *  - Unit: Ppm
```

Comment:
Do some of these units need an additional explanation?
E.g. 
Unit: Ppm (parts per million)

We also have ppb, Kvah, Va, Klmh, Klxh.

Reply:
I guess this will also be part of the cleanup task :)